frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -183,10 +199,12 @@ const EditRecipeForm = ({ history }) => {
         ingredients={ingredients}
         setIngredients={setIngredients}
       />
+      <div className="errorMsg">{ingredients.length <= 1 ? ingredientsError : ''}</div>
```

Comment:
I think you can just do `{ingredientsError}`, because the `ingredientsError ` is an empty string by default.